package.json

```diff
@@ -8,6 +8,7 @@
     "start:ts": "./node_modules/.bin/ts-node index.ts",
     "watch": "./node_modules/.bin/nodemon -e ts --exec npm start",
     "lint": "./node_modules/.bin/tslint -c tslint.json 'src/**/*.ts'",
+    "precommitLint": "./node_modules/.bin/tslint -c tslint.json ",
```

Comment:
i will not create it as a npm script, why not simply run it from the `pre-commit` script?

Reply:
It should be at the pre-commit stage but, to have the option in an `npm` is a `good-to-have` to me